CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Ignore frequent trace spans
+
```

Comment:
You will also need to update the end of this file where it says

```
[Unreleased]: https://github.com/itchysats/itchysats/compare/0.5.1...HEAD
[0.5.0]: https://github.com/itchysats/itchysats/compare/0.5.0...0.5.1
```

Note there was a typo which was fixed on master.